os/tools/compression/src/compress.c

```diff
@@ -228,6 +231,8 @@ static void compress_file(int block_size, int type, char *in_file, char *out_fil
 		}
 	}
 
+	exit_code = 0; /* Compression is successful */
```

Comment:
same as 0

Reply:
I didn't understand. Please correct me If I am wrong. exit 0 indicates that the execution was successful. Should I add that the execution was successful in the comment.

Reply:
Yes. We also know nomarlly 0 is indicates the success and other variables are failure.
But it is better to add comment for better understanding.

Reply:
What you understood was correct :)